.github/workflows/publish_to_test.yaml

```diff
@@ -0,0 +1,31 @@
+name: Publish To Test
+
+on:
+  workflow_dispatch:
+  
+env:
+  AWS_ACCESS_KEY_ID: ${{ secrets.TEST_AWS_ACCESS_KEY_ID }}
+  AWS_SECRET_ACCESS_KEY: ${{ secrets.TEST_AWS_SECRET_ACCESS_KEY }}
+  AWS_DEFAULT_REGION: eu-central-1
+  AWS_DISTRIBUTION_ID: ${{ secrets.TEST_AWS_DISTRIBUTION_ID }}
+  AWS_BUCKET_NAME: ${{ secrets.TEST_AWS_BUCKET_NAME }}
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Setup node.js
+      uses: actions/setup-node@v1
+      with:
+        node-version: 16.4
+
+    - name: Build website
+      run: npm install gatsby-cli && npm run build
```

Comment:
Hm, actually, it shouldn't be needed...

```cli
$ which gatsby-cli
gatsby-cli not found

$ which gatsby
gatsby not found

$ npm run build

> alephium-website@1.0.0 build
> gatsby build

success open and validate gatsby-configs, load plugins - 0.791s
...
```

Reply:
Something changed since we commit'ed the publish.yaml workflow, but yes the `npm install gatsby-cli` step doesn't seems to be required anymore.
Fixing.

Reply:
Hm, I am still sceptical about this 🤔 At the moment there is no command to install the npm packages. Neither `npm ci` nor `npm install` is present. I wonder how this workflow installs the required packages...

Reply:
AH! nvm! Found the answer by checking the code of the setup-node action: https://github.com/actions/setup-node

It includes the `npm install` step.

Reply:
That's the magic of the step 
```
    - name: Setup node.js
      uses: actions/setup-node@v1
```